adapters/triplelift_native/triplelift_native.go

```diff
@@ -53,7 +69,21 @@ func processImp(imp *openrtb2.Imp) error {
 	if tlext.InvCode == "" {
 		return fmt.Errorf("no inv_code specified")
 	}
-	imp.TagID = tlext.InvCode
+
+	if ext.Data != nil {
+		extData = *ext.Data
+	}
+
+	if extData.TagCode != "" {
+		if siteCopy.Publisher.Domain == "msn.com" {
+			fmt.Println(extData.TagCode)
+			imp.TagID = extData.TagCode
+		} else {
+			imp.TagID = tlext.InvCode
+		}
```

Comment:
This conditional is missing test coverage. You might want to consider adding a JSON test to cover this case.

Reply:
optional-params.json accounts for this case

Reply:
This `else` does not appear to be covered. The outer `else` with the same assignment just after this block is covered though.